### PR TITLE
Add link to the code of DeepTyper paper

### PR DESCRIPTION
**Pull request by bzz**
Add link to the code for a DeepTyper paper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ A curated list of awesome research papers, datasets and software projects devote
...
 * <img src="badges/12-pages-gray.svg" alt="12-pages" align="top"> [Suggesting Accurate Method and Class Names](http://homepages.inf.ed.ac.uk/csutton/publications/accurate-method-and-class.pdf) - Miltiadis Allamanis, Earl T. Barr, Christian Bird, Charles Sutton, FSE 2015.
 * <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [Mining Source Code Repositories at Massive Scale using Language Modeling](http://homepages.inf.ed.ac.uk/csutton/publications/msr2013.pdf) - Miltiadis Allamanis, Charles Sutton, MSR 2013.
-* <img src="badges/11-pages-gray.svg" alt="11-pages" align="top"> [Deep Learning Type Inference](http://vhellendoorn.github.io/PDF/fse2018-j2t.pdf) - Vincent J. Hellendoorn, Christian Bird, Earl T. Barr and Miltiadis Allamanis, FSE 2018.
+* <img src="badges/11-pages-gray.svg" alt="11-pages" align="top"> [Deep Learning Type Inference](http://vhellendoorn.github.io/PDF/fse2018-j2t.pdf) - Vincent J. Hellendoorn, Christian Bird, Earl T. Barr and Miltiadis Allamanis, FSE 2018. [code](https://github.com/DeepTyper/DeepTyper)
```
Comment (m09) [inline README.md]:
```suggestion
* <img src="badges/11-pages-gray.svg" alt="11-pages" align="top"> [Deep Learning Type Inference](http://vhellendoorn.github.io/PDF/fse2018-j2t.pdf) - Vincent J. Hellendoorn, Christian Bird, Earl T. Barr and Miltiadis Allamanis, FSE 2018. [Code](https://github.com/DeepTyper/DeepTyper).
```

**Review by m09 — COMMENTED** [review]
Minor formatting edit.

File: README.md (hunk 1 above)
Comment (bzz) [reply README.md]: it's just that `. [video](....)` link in here https://github.com/src-d/awesome-machine-learning-on-source-code/blob/96ee8af0a07edab23634661798b70aef10dae9b4/README.md#program-repair-and-bug-detection was not capitalized

Comment (m09) [reply README.md]: Oh right, good point. I guess my preferred solution is to update both to capitalized form + ending dot but if you prefer to keep as is it's also fine for me it's really a minor detail :)

**Review by m09 — APPROVED** [review]

**Comment by bzz** [comment]
ping @EgorBu

**Review by EgorBu — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**